### DISCUSSION TITLE
Fastify with Sandpack

### DISCUSSION THREAD

**Discussion by andysm1th** [fastify/fastify#4604] [Q&A] [answered]
I'm currently trying to run Fastify inside of [Sandpack](https://sandpack.codesandbox.io/docs) but I'm having some dns related issues and wondered if anyone could help.

I've added the required dependencies to run the project but get this error:

```
/fastify/server.js: Could not find dependency: 'dns' relative to '/node_modules/fastify/lib/server.js' (1:0)
```

What I don't understand is that Fastify (or the file I'm using) does not require the dns module.

But when I add the required 'dns' package I get the following errors:

```
/fastify/server.js: Error: ENOENT: No such file or directory., '/etc/hosts' (1:0)
/fastify/server.js: Error: ENOENT: No such file or directory., '/etc/resolv.conf' (1:0)
```

Does anyone have any idea why this could be happening or point me in the right direction?

I am able to run Fastify in WebContainers so this could be related to Sandpack's proprietary Node implementation: https://sandpack.codesandbox.io/docs/advanced-usage/nodebox

Cheers!

@simoneb

**Comment by andysm1th** [comment, marked as answer]
This was an issue relating to my setup of Sandpack and wasn't related to the Fastify project

**Reply by simoneb** [reply]
We've got it up running on https://github.com/nearform/fastify-playground-sandpack
